src/button/stories/Button.stories.tsx

```diff
@@ -95,7 +95,7 @@ export const CustomLoadingElement = base({
 const iconButtonBase = storyTemplate<IconButtonProps>(
   args => {
     return (
-      <IconButton aria-label="picture" {...args}>
+      <IconButton as="img" src="" aria-label="picture" {...args}>
         <SearchIcon />
       </IconButton>
```

Comment:
<img width="1080" alt="Screenshot 2021-02-17 at 2 21 36 PM" src="https://user-images.githubusercontent.com/35374649/108179388-91e26d80-712b-11eb-9ada-9b7f32e3de49.png">

Reply:
Right ! that was an example I am testing.
Never intended to push. I will remove that code.